Swift Vocabulary/Swift Vocabulary/WordsTableViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  WordsTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by Bryan Cress on 5/12/20.
+//  Copyright © 2020 Bryan Cress. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+    
+    var vocabWords: [VocabularyWord] = [VocabularyWord(word: "Segue", definition: "A transition from one story board scene to another"),
```

Comment:
Nice job making it easy to read the array, rather than just writing the whole thing on a single line.